cfme/tests/middleware/test_middleware.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+from utils.browser import ensure_browser_open
+from cfme.login import login_admin
+from cfme.fixtures import pytest_selenium as sel
+
+
+@pytest.fixture
+def webDriver(request):
+    ensure_browser_open()
+    login_admin()
+
+    def closeSession():
+        """ Place Holder for after-test action """
+    request.addfinalizer(closeSession)
+
+    return
+
+
+def test_middleware():
+    sel.force_navigate('middleware')
```

Comment:
And actually, you should not import sel, but use it from `pytest.sel` so instead of importing and writing `sel.force_navigate(...)` you should do `pytest.sel.force_navigate(...)`.